packages/flutter_tools/pubspec.yaml

```diff
@@ -13,14 +13,17 @@ dependencies:
   crypto: 0.9.2
   file: ^0.1.0
   http: ^0.11.3
+  intl: '>=0.12.4+2 <0.13.0'
+  json_rpc_2: ^2.0.0
   json_schema: ^1.0.3
   mustache4dart: ^1.0.0
   package_config: ^0.1.3
   path: ^1.3.0
   pub_semver: ^1.0.0
   stack_trace: ^1.4.0
-  yaml: ^2.1.3
+  web_socket_channel: ^1.0.0
```

Comment:
I'm hoping to tie into the observatory from `flutter run` shortly, at which point I'll probably use the https://github.com/dart-lang/vm_service_drivers/tree/master/dart library. @yjbanov is using the https://github.com/dart-lang/vm_service_client library in flutter_driver. I don't know that this is actionable in this PR, but at some point we should look at consolidating.